src/mapgen/mapgen_valleys.cpp

```diff
@@ -109,7 +119,7 @@ MapgenValleysParams::MapgenValleysParams():
 	np_filler_depth       (0.0,   1.2,  v3f(256,  256,  256),  1605,  3, 0.5,  2.0),
 	np_inter_valley_fill  (0.0,   1.0,  v3f(256,  512,  256),  1993,  6, 0.8,  2.0),
 	np_inter_valley_slope (0.5,   0.5,  v3f(128,  128,  128),  746,   1, 1.0,  2.0),
-	np_rivers             (0.0,   1.0,  v3f(256,  256,  256),  -6050, 5, 0.6,  2.0),
+	np_rivers             (0.0,   1.0,  v3f(256,  256,  256),  -6050, 5, 0.6,  2.0, NOISE_FLAG_EASED),
```

Comment:
Eased 3D noise is intensive compared to normal 3D noise, just so you know. However i'm not insisting on non-eased noise. But with 'canyons' enabled valleys mapgen will become more intensive than v7.

Reply:
Non-eased 3D noise doesn't look nice for what I've tested. And this makes it more consistent with the 2D version that is eased by default.